sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -127,36 +132,6 @@ def _urljoin(base_url, stub_url):
     parsed = parsed._replace(path=parsed.path.rstrip("/") + "/" + stub_url)
     return parsed.geturl()
 
-
-class _HTTPSerializer(HTTPConnection, object):
```

Comment:
I created a shared file: azure.core.utils._pipeline_transport_rest_shared, where I moved all this code, so it can be shared between the old and new requests